lib/pltr/v2/slate_serializers/to_html.js

```diff
@@ -18,6 +18,12 @@ export const rtfToHTML = (arrayBuffer) => {
         for (let j = 0; j < nodeList.length; ++j) {
           result.push(nodeList[j])
         }
+
+        if (!nodeList.length) {
```

Comment:
@jeanawhou, could this create a whole bunch of empty paragraphs?
What if we moved it outside the outer loop and checked whether `result` is empty?
If the `result` is empty, we can add a node to the result list exactly once.

Reply:
@Quiescent this only will create a new span if the nodeList is empty, not the entire result (1st image). I added this check so that if the user didn't put any description for the custom attribute but has title or it is blank it will have a placeholder of 3 dots and it will not break custom attribute and get the next text (which is Role see 2nd image) 😅 

Each nodeList will be a custom attribute and then its description consecutively 
So the nodeLists array there on the local values of the first image is 

first nodeList [0] = **title**
first nodeList [1] = **first custom attribute**
next nodeList **first custom attribute description**

next nodeList **2nd custom attribute**
next nodeList **2nd custom attribute description**, and so on...
What do you think ed?

<img width="1493" alt="Screen Shot 2022-03-15 at 4 54 27 PM" src="https://user-images.githubusercontent.com/19387007/158342250-5429d421-841a-4cb3-b10a-816f07acb50a.png">

<img width="908" alt="image" src="https://user-images.githubusercontent.com/19387007/158342835-f2e668df-35eb-4390-a368-8e9be55cdc21.png">

Reply:
Sorry this is the right nodeLists for that three little pigs i was uploading 
<img width="1497" alt="image" src="https://user-images.githubusercontent.com/19387007/158346111-6559e5bf-aec4-46aa-96a6-d67c1ba9492a.png">

Reply:
Right!  Thanks for explaining.
I think that's fine.  Let's see what Troy and others think about how it looks in practice.